cli/tests/test_git_fetch.rs

```diff
@@ -14,46 +14,41 @@
 use std::path::Path;
 
 use test_case::test_case;
+use testutils::git;
 
 use crate::common::CommandOutput;
 use crate::common::TestEnvironment;
 
-fn add_commit_to_branch(git_repo: &git2::Repository, branch: &str) -> git2::Oid {
-    let signature = git2_signature();
-    let mut tree_builder = git_repo.treebuilder(None).unwrap();
-    let file_oid = git_repo.blob(branch.as_bytes()).unwrap();
-    tree_builder
-        .insert("file", file_oid, git2::FileMode::Blob.into())
-        .unwrap();
-    let tree_oid = tree_builder.write().unwrap();
-    let tree = git_repo.find_tree(tree_oid).unwrap();
-    git_repo
-        .commit(
-            Some(&format!("refs/heads/{branch}")),
-            &signature,
-            &signature,
-            "message",
-            &tree,
-            &[],
-        )
-        .unwrap()
-}
-
-fn git2_signature() -> git2::Signature<'static> {
-    git2::Signature::new("Some One", "some.one@example.com", &git2::Time::new(0, 0)).unwrap()
+fn add_commit_to_branch(git_repo: &gix::Repository, branch: &str) -> gix::ObjectId {
+    git::add_commit(
+        git_repo,
+        &format!("refs/heads/{branch}"),
+        branch,
```

Comment:
nit: I would add `branch, // filename` comment here because `branch` is confusing in this context.